homePositions:
  - 1.570796        # shoulder_pan_joint
  - -2.35619449019  # shoulder_lift_joint
  - 2.35619449019   # elbow_joint
  - -3.14159        # wrist_1_joint
  - -1.570796       # wrist_2_joint
  - 0.0             # wrist_3_joint

bedOrigin:
  - -0.113007152    # x
  - 0.494493063     # y
  - 0.554776374     # z

# Camera Intrinsic Calibration parameters
pointPerSide: 4
pointPerHeight: 2     

# Handeye Calibration parameters
numPointsPerVertArc: 10 
numVertArc: 18 
numRadius: 2 
    
# Laser Cam calibration parameters
numLinePerPlane: 4 
numPlaneAngles: 3 
numPlanePerCircle: 16 

# Scanning parameters
rightPositions:
  - -0.51
  - -1.65
  - -1.71
  - 0.13
  - -0.6
  - 0